www/include/configuration/configCentreonBroker/formCentreonBroker.php

```diff
@@ -168,15 +171,9 @@
     ));
     $tpl->assign('config_id', 0);
 } elseif ($id !== 0) {
+    $id = $_GET['id'];
     $tpl->assign('config_id', $id);
-    $defaultBrokerInformation = getCentreonBrokerInformation($id);
-    if (!isset($defaultBrokerInformation['log_core'])) {
-        $defaultBrokerInformation = array_merge(
-            $defaultBrokerInformation,
-            $defaultLog
-        );
-    }
-    $form->setDefaults($defaultBrokerInformation);
+    $form->setDefaults(getCentreonBrokerInformation($id));
```

Comment:
$defaultLog was not introduced until the 21.04, resulting in an error on the array_merge